import_activity_events.py

```diff
@@ -59,7 +65,7 @@
 """
 
 Q_CREATE_EVENTS_TABLE = """
-    CREATE TABLE IF NOT EXISTS activity_events (
+    CREATE TABLE IF NOT EXISTS activity_events_{sample_rate} (
```

Comment:
Renaming the main `activity_events` table will break all the existing re:dash graphs.  It's probably much uglier in code, but can we keep `activity_events` for the full export and only add suffixes to the sampled tables?